kart/point_cloud/mosaic_util.py

```diff
@@ -0,0 +1,38 @@
+import logging
+
+from tempfile import NamedTemporaryFile
+from kart.subprocess_util import check_output
+
+
+L = logging.getLogger("kart.point_cloud")
+
+
+def write_vpc_for_directory(directory_path):
+    """
+    Given a folder containing some LAZ/LAS files, write a mosaic file that
+    combines them all into a single Virtual Point Cloud (VPC). The VPC will
+    contain references to the tiles, rather than replicating their contents.
+    """
+    vrt_path = directory_path / f"{directory_path.name}.vpc"
+
+    tiles = [str(p) for p in directory_path.glob("*.copc.laz")]
+    if not tiles:
+        vrt_path.unlink(missing_ok=True)
+        return
+
+    with NamedTemporaryFile("w+t", suffix=".kart_tiles", encoding="utf-8") as tile_list:
```

Comment:
Haven't tested it, but whenever I see this kind of thing I think "this will probably break on windows..." but it depends, windows can certainly share an open file between two processes if the processes feel like it, but seems like sometimes they don't.
Anyway if it does break on windows, this will be why.